console/console-init/ui/src/modules/project/ProjectPage.tsx

```diff
@@ -149,6 +158,49 @@ export default function ProjectPage() {
       }
     }
   };
+  const handleOnChangeEnable = (projects: IProject[], toEnable: boolean) => {
+    let iotQueryVariables: Array<{ name: string; namespace: string }> = [];
+    selectedProjects.forEach(
+      (project: IProject) =>
+        project.name &&
+        project.namespace &&
+        project.projectType === ProjectTypes.IOT &&
+        iotQueryVariables.push({
+          name: project.name,
+          namespace: project.namespace
+        })
+    );
+    if (iotQueryVariables.length > 0) {
+      const queryVariable = {
+        a: iotQueryVariables,
+        status: toEnable
+      };
+      setToggleIoTProjectQueryVariables(queryVariable);
+    }
+  };
+
+  const onToggleAll = (action: string) => {
+    const toEnable = action === "enable" ? true : false;
+    const filteredProjects = selectedProjects.filter(
+      project =>
+        project.projectType === ProjectTypes.IOT &&
+        project.isEnabled === !toEnable
+    );
+    let projectActionValue = toEnable ? "Enable" : "Disable";
```

Comment:
projectStatus name would be more better than projectActionValue.

Reply:
this is not the status of the project, this is the action from the kebab dropdown